app/src/main/res/layout/fragment_register.xml

```diff
@@ -1,8 +1,91 @@
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/_24sdp"
+        android:layout_marginTop="@dimen/_54sdp"
+        android:layout_marginEnd="@dimen/_24sdp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tv_header_sign_up">
+
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/et_email"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_28sdp"
+                android:layout_marginTop="@dimen/_28sdp"
+                android:layout_marginEnd="@dimen/_28sdp"
+                android:layout_marginBottom="@dimen/_20sdp"
+                android:fontFamily="@font/nunito"
+                android:hint="@string/email"
+                android:singleLine="true"
+                android:textColorHint="@color/abu"
+                android:textSize="@dimen/_13ssp" />
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/et_password"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_28sdp"
+                android:layout_marginEnd="@dimen/_28sdp"
+                android:layout_marginBottom="@dimen/_20sdp"
+                android:fontFamily="@font/nunito"
+                android:hint="@string/password"
+                android:singleLine="true"
+                android:textColorHint="@color/abu"
+                android:textSize="@dimen/_13ssp" />
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/et_username_signUp"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/_28sdp"
+                android:layout_marginEnd="@dimen/_28sdp"
+                android:layout_marginBottom="@dimen/_20sdp"
+                android:fontFamily="@font/nunito"
+                android:hint="@string/username"
+                android:singleLine="true"
+                android:textColorHint="@color/abu"
+                android:textSize="@dimen/_13ssp" />
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/btn_sign_up"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/_48sdp"
+                android:layout_margin="@dimen/_24sdp"
+                android:fontFamily="@font/nunito"
+                android:singleLine="true"
```

Comment:
ki singleline jare rak kanggo kan? mbuk nggo ning edit text to?

Reply:
iyo,loh rk ngaruh kah nek ng editText??

Reply:
nek ning edit text ngaruh, tapi kan nek ning button rak kanggo kan

Reply:
lohh mosok ng button ono??walah janck wkwkw